### PR TITLE
[DA] clarify next day threshold value

### PR DESCRIPTION
**Pull request by dakala**
Issue: https://github.com/fullcalendar/fullcalendar/issues/4541

### CODE REVIEW

**Comment by arshaw** [comment]
@dakala , the current docs are actually correct. in V3, the default value was 9am, but in V4 it was changed to midnight.

The v3 docs are correct but I simply want us to put more emphasis on the settings. The setting  sometimes makes the event appear to end one day earlier.

**Comment by dakala** [comment]
@arshaw I've reverted the initial changes  to v3. What do you think about the latest commit to v4? Apart from clarifying the setting, I think the example needs to be clearer on where it goes. 

Once the v4 is done, we can decide whether any changes to v3 docs are required.

File: _docs-v4/event-display/nextDayThreshold.md [hunk 1]
```diff
@@ -13,13 +13,72 @@ Only affects timed events that appear on whole-days. Whole-day cells occur in [D
...
-{ start: '2014-02-04T20:00:00', end: '2014-02-05T02:00:00' }
-// goes from 8pm to 2am the next day
+import { Calendar } from '@fullcalendar/core';
+import dayGridPlugin from '@fullcalendar/daygrid';
+import timeGridPlugin from '@fullcalendar/timegrid';
+
+document.addEventListener('DOMContentLoaded', function() {
+  var calendarEl = document.getElementById('calendar');
+
+  var calendar = new Calendar(calendarEl, {
```
Comment (arshaw) [inline _docs-v4/event-display/nextDayThreshold.md]: this example is really good, but there's a lot to it that don't directly relate to the point you're trying to make. that's why I like to write examples like this...

```js
var calendar = new Calendar(calendarEl, {
  nextDayThreshold: '09:00:00',
  events: [    
    {
      // Goes from 8pm to 2am the next day.
      title: 'Event 1',
      start: '2014-02-04T20:00:00',
      end: '2014-02-05T02:00:00'
    }
  ]
  // other options...
});
```

what do you think?

Comment (dakala) [reply _docs-v4/event-display/nextDayThreshold.md]: Yeah, agreed. The examples need to be shorter and clutter-free. 

This is the first step. Next, I'll go through the tickets in fullcalendar/fullcalendar#3205 again as well as the docs to see where we can add a bit of clarity or link to this nextDayThreshold or  event-parsing page.

**Comment by dakala** [comment]
@arshaw I think all relevant v4 instances are now covered. Also, see https://github.com/fullcalendar/fullcalendar-site-static/pull/80 please.

**Pull request merged**